robotbase/src/main/java/com/team2357/lib/arduino/ArduinoSensor.java

```diff
@@ -0,0 +1,39 @@
+package com.team2357.lib.arduino;
+
+public class ArduinoSensor extends ArduinoUSBController{
+
+    private String m_deviceName;
+
+    public ArduinoSensor(String ttyDevice, String deviceName) {
+        super(ttyDevice);
+        
+        m_deviceName = deviceName;
+    }
+
+    public boolean getIntakeValue() {
+        boolean value = false;
+        if (this.isConnected()){
+            value = !this.getDeviceFieldBoolean(m_deviceName, "state");
```

Comment:
Instead of m_deviceName we should be using a constant to refer to the specific device name.